python/ray/tests/test_multi_node.py

```diff
@@ -426,7 +426,7 @@ def test_run_driver_twice(ray_start_regular):
 def train_func(config, reporter):  # add a reporter arg
     for i in range(2):
         time.sleep(0.1)
-        reporter(timesteps_total=i, mean_accuracy=i+97)  # report metrics
+        reporter(timesteps_this_iter=i, mean_accuracy=i+97)  # report metrics
```

Comment:
this doesn't need to be changed right?

Reply:
Sorry, this change is wrong. Given it increments the timesteps_total manually, this should be reverted. See my other reply about timesteps_total and timesteps_this_iter.